pysumma/distributed.py

```diff
@@ -0,0 +1,141 @@
+from copy import deepcopy
+from distributed import Client, get_client
+from typing import List, Dict
+import os
+from pathlib import Path
+import pandas as pd
+import time
+import numpy as np
+import xarray as xr
+
+from .file_manager import FileManager
+from .simulation import Simulation
+from .utils import ChainDict, product_dict
+
+OMP_NUM_THREADS = int(os.environ.get('OMP_NUM_THREADS', 1))
+
+class Distributed(object):
+    '''
+    Distributed objects represent SUMMA configurations where
+    there are multiple GRU/HRU which are expected to be run
+    in parallel.
+
+    Currently only supports GRU based parallelization.
+    '''
+
+    def __init__(self, executable: str, filemanager: str,
+                 num_workers: int=1, threads_per_worker: int=OMP_NUM_THREADS,
+                 chunk_size: int=None, num_chunks: int=None, scheduler: str=None):
```

Comment:
Can you explain what the chunk_size and num_chunks are?

Reply:
Added docstring